test/sql/copy/csv/auto/test_sample_size.test

```diff
@@ -5,17 +5,9 @@
 require vector_size 512
 
 # CSV file with very sparse column
-statement ok
+statement error
 CREATE TABLE test AS SELECT * FROM read_csv_auto ('test/sql/copy/csv/data/auto/issue_811.csv', SAMPLE_SIZE=1024);
```

Comment:
Sounds weird, but I believe that this passing was actually a bug.

The reason this was not failing before was because of this line:
https://github.com/duckdb/duckdb/pull/6756/files#diff-ab757987c109f9f46a7cd0876d1971730997734da4abe4ba27402738f457771cL816
This line would cause the sniffer to detect a NULL type but would be exchanged to varchar, even though the column in question is 99.9% Null and with the set sample size, the sniffer only really goes through these null lines.